pootle/apps/pootle_store/models.py

```diff
@@ -18,7 +18,7 @@
 from django.contrib.auth import get_user_model
 from django.core.exceptions import ObjectDoesNotExist
 from django.core.urlresolvers import reverse
-from django.db import models, transaction
+from django.db import models
```

Comment:
Can you please directly land the adding missing imports and remove unused imports changes?